rust-code-analysis-cli/src/main.rs

```diff
@@ -101,6 +102,23 @@ fn act_on_file(path: PathBuf, cfg: &Config) -> std::io::Result<()> {
             let path = cfg.path.clone();
             action::<Metrics>(&language, source, &path, pr, cfg)
         }
+    } else if cfg.wmc {
+        if language == LANG::Java {
```

Comment:
Why is it Java only ?
What about js, c++, potentially C# ?

Reply:
I didn't realize that line was useless. I removed it. Thank you for pointing it out!